pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -388,6 +388,22 @@ impl Layer {
             })
     }
 
+    /// Get all key/values in the layer. Should be replaced with an iterator-based API in the future.
+    pub(crate) async fn get_all(
```

Comment:
Please use same method name as the one it ultimately delegates to (load_key_values).